bin/index.js

```diff
@@ -11,44 +11,76 @@ const jsonFormatter = require('../lib/formatters/json');
 const yamlFormatter = require('../lib/formatters/yaml');
 const SVG = require('../lib/svg');
 
-// Declare the app
-const app = (filePath, options) => {
+const parseStdin = () => {
+  return new Promise((resolve, reject) => {
+    let data = '';
+    
+    process.stdin.on('data', (chunk) => {
+      data += chunk;
+    });
+
+    process.stdin.on('error', (error) => {
+      reject(error);
```

Comment:
In `parseFile`, the reject method receives an `String`, so I think we should display a proper message here too. Something like:

```
reject(`There was an error parsing the STDIN data: ${error}`);
```